modules/standard/Random.chpl

```diff
@@ -21,19 +21,29 @@
    Support for pseudorandom number generation
 
    This module defines an abstraction for a stream of pseudorandom numbers,
-   :class:`~Random.RandomStreamInterface`, supporting methods to get the next random
-   number in the stream (:proc:`~Random.RandomStreamInterface.getNext`), to
-   fast-forward to a specific value in the stream
+   :class:`~Random.RandomStreamInterface`, supporting methods to get the next
+   random number in the stream (:proc:`~Random.RandomStreamInterface.getNext`),
+   to fast-forward to a specific value in the stream
    (:proc:`~Random.RandomStreamInterface.skipToNth` and
-   :proc:`~Random.RandomStreamInterface.getNth`), or to fill an array with random
-   numbers in parallel (:proc:`~Random.RandomStream.fillRandom`).  The module
-   also provides a standalone convenience function, :proc:`fillRandom` that can
-   be used to fill an array with random numbers in parallel without manually
-   creating a :class:`RandomStream` object.
+   :proc:`~Random.RandomStreamInterface.getNth`), to iterate over random values
+   :proc:`~Random.RandomStreamInterface.iterate` (possibly in parallel), or to
+   fill an array with random numbers in parallel
+   (:proc:`~Random.RandomStream.fillRandom`).  The module also provides a
+   standalone convenience function, :proc:`fillRandom` that can be used to fill
+   an array with random numbers in parallel without manually creating a
+   :class:`RandomStream` object.
 
    Use :proc:`makeRandomStream` or the constructor for a specific RNG
    implementation to get a RandomStream.
 
+   .. note::
+
+      We expect in the future for RandomStream to be the default and for both
+      NPBRandomStream and PCGRandomStream to be available. In this version,
+      RandomStream is the default and refers to a PCG implementation. The
+      function :proc:`makeRandomStream` is available to avoid compatibility
+      problems from this naming change.
```

Comment:
Post-feature freeze comment: This note isn't particularly clear to me.  The main thing that isn't clear is that it suggests that both in the future and currently, RandomStream is the default, suggesting there's no difference there. Also, default what?